app/domain/authentication/authn_jwt/validate_input.rb

```diff
@@ -0,0 +1,27 @@
+require 'command_class'
+
+module Authentication
+  module AuthnJwt
+
+    ValidateInput ||= CommandClass.new(
+      dependencies: {
+        jwt_validate_body: Authentication::AuthnJwt::ValidateRequestBody.new
+      },
+      inputs: %i[authentication_parameters]
+    ) do
+
+      def call
+        jwt_validate_body
+      end
+
+      private
+
+      def jwt_validate_body
```

Comment:
please call it `validate_request_body`

Reply:
or even better `validate_body_is_jwt`. this way we can easily add more check in the future.